modules/pivot-table/main/index.coffee

```diff
@@ -56,7 +56,8 @@ class PivotTable extends hx.EventEmitter
       cellRender: (data, element, isHead, column) ->
         hx.select(element).text(data)
       useResponsive: true
-      data: undefined
+      data: undefined,
+      fullWidth: false
```

Comment:
this is fine for our usecase, but maybe it should be set to "true" ? As you almost always want it